schema/openeox_shell_json_schema.json

```diff
@@ -0,0 +1,70 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://docs.oasis-open.org/openeox/tbd/openeox_shell_json_schema.json",
```

Comment:
Using path separators:
`https://docs.oasis-open.org/openeox/tbd/schema/shell.json`

Reply:
addressed in [b57311f](https://github.com/oasis-tcs/openeox/pull/10/commits/b57311f3895a5208794ebec25f72fbf200fb4d3e)